orchestration/Tests/UnitTests/ModuleInstanceDeployment.Tests.ps1

```diff
@@ -17,7 +17,7 @@ Describe  "Module Instance Deployment Orchestrator Unit Test Cases" {
             $defaultWorkingDirectory = "$ScriptPath/../../../../";
             $ArchetypeDefinitionPath = "$ScriptPath/../../../../archetypes/shared-services/2.0/archetypeDefinition.json";
             $archetypeInstanceJson = `
-                Get-ArchetypeInstance `
+                Build-ConfigurationUsingFile `
```

Comment:
Do you plan to add a function called: Build-ConfigurationNotUsingFile? I don't understand the function name

Reply:
Renamed to Build-ConfigurationObject.